recipes/googletest/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.7.0" %}
+{% if linux %}
+    {% set libext = ".so" %}
+{% else %}
+    {% set libext = ".dylib" %}
+{% endif %}
+
+package:
+  name: googletest
+  version: {{ version }}
+
+source:
+  fn: release-{{ version }}.tar.gz
+  url: https://github.com/google/googletest/archive/release-{{ version }}.tar.gz
+  sha256: f73a6546fdf9fce9ff93a5015e0333a8af3062a152a9ad6bcb772c96687016cc
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python <3  # tests are Python 2k only :(
```

Comment:
The `skip` change above will make this unnecessary. Using `skip` is the preferred way for handling platform, Python, or NumPy constraints. Other constraints are fine to handle this way.